sunbird-docker/sunbird_player.env

```diff
@@ -0,0 +1,20 @@
+sunbird_port=3000
+sunbird_content_player_url=https://spike.open-sunbird.org/api/
```

Comment:
are these correct, this needs to be the name of the service which will be setup in the docker overlay network